cookbook/case_studies/ml_training/house_price_prediction/multiregion_house_price_predictor.py

```diff
@@ -1,19 +1,35 @@
 """
-Predicting House Price in Multiple Regions with XGBoost and Dynamic Workflows
-------------------------------------------------------------------------------
+Predicting House Price in Multiple Regions Using XGBoost and Dynamic Workflows
+-------------------------------------------------------------------------------
+`XGBoost <https://xgboost.readthedocs.io/en/latest/>`__ is an optimized distributed gradient boosting library designed to be efficient, flexible, and portable. 
+It uses `gradient boosting <https://en.wikipedia.org/wiki/Gradient_boosting>`__ technique to implement Machine Learning algorithms.
```

Comment:
Could we remove this because we are explaining what XGBoost is in the other Python file?